src/SODAComponents/GitHubScreen/GitHubScreen.jsx

```diff
@@ -13,7 +13,7 @@ import ContributorsGeoDistribution from "./ContributorsGeoDistribution/Contribut
 import CompaniesScatter from "./CompaniesScatter/CompaniesScatter.jsx";
 
 // var BASE_URL = "http://192.168.3.16:8081";
-var BASE_URL = "http://nzcer.cn:8081";
+var BASE_URL = "https://nzcer.cn:8081";
```

Comment:
What is this URL used for? I got errors while visiting it.

Reply:
This is for the Data Screen toggled by ` key. The data now is provided by a java service, I think it should be stored as files on oss. I'll work on this.

Reply:
Yes, it should be on OSS too, I can help with this too later.